### PR TITLE
Fix Julia documentation.

### PR DESCRIPTION
**Pull request by Yashwants19**
This PR is created to fix Julia documentation as discussed with @rcurtin. The `info`, `help`, `version` and `copy_all_inputs` inputs aren't actually a part of the Julia binding as printed in the binding documentation.

### CODE REVIEW

File: .ci/macos-steps.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ steps:
     set -e
     sudo xcode-select --switch /Applications/Xcode_10.1.app/Contents/Developer
     unset BOOST_ROOT
-    pip install cython numpy pandas zipp
+    pip install cython numpy pandas zipp configparser
```
Comment (rcurtin) [inline .ci/macos-steps.yaml:17]: I don't have a problem adding this, but are you sure it's needed?

File: src/mlpack/bindings/python/print_doc_functions_impl.hpp [hunk 2]
```diff
@@ -240,7 +240,8 @@ inline std::string ProgramCall(const std::string& programName)
   bool first = true;
   for (auto it = parameters.begin(); it != parameters.end(); ++it)
   {
-    if (!it->second.input || it->second.persistent)
+    if (!it->second.input || (it->second.persistent &&
+        it->second.name != "verbose" && it->second.name != "copy_all_inputs"))
```
Comment (rcurtin) [inline src/mlpack/bindings/python/print_doc_functions_impl.hpp:244]: Ah, nice, thanks for fixing this for Python too.  But `copy_all_inputs` is Python-specific, so we should leave that in. :+1:

File: src/mlpack/bindings/julia/print_doc_functions_impl.hpp [hunk 3]
```diff
@@ -475,7 +475,8 @@ inline std::string ProgramCall(const std::string& programName)
   size_t nonreqInputs = 0;
   for (auto it = parameters.begin(); it != parameters.end(); ++it)
   {
-    if (it->second.input && !it->second.required)
+    if (it->second.input && !it->second.required &&
+       (it->second.name == "verbose" ||  !it->second.persistent))
```
Comment (rcurtin) [inline src/mlpack/bindings/julia/print_doc_functions_impl.hpp:479]:
```suggestion
       (it->second.name == "verbose" || !it->second.persistent))
```

File: .ci/macos-steps.yaml (hunk 1 above)
Comment (Yashwants19) [reply .ci/macos-steps.yaml]: I think so this is already added I don't why this is happening after revert my local changes.
I am really sorry about this but this is resolved after rebasing the branch.

File: src/mlpack/bindings/python/print_doc_functions_impl.hpp (hunk 2 above)
Comment (Yashwants19) [reply src/mlpack/bindings/python/print_doc_functions_impl.hpp]: Removed. :)

File: src/mlpack/bindings/julia/print_doc_functions_impl.hpp (hunk 3 above)
Comment (Yashwants19) [reply src/mlpack/bindings/julia/print_doc_functions_impl.hpp]: Updated.

**Comment by Yashwants19** [comment]
Hi @rcurtin I have updated `HISTORY.md` as suggested.
Thank You.

**Comment by rcurtin** [comment]
Awesome, thank you! :+1: Let's make sure the build passes, then I'll approve and merge. :)

**Comment by rcurtin** [comment]
When 3.3.1 is released, this should be propagated to the stable documentation pages.  Tonight, the website rebuild should update the mlpack-git documentation.  Thanks! :+1:

**Pull request merged**